packages/salesforce/test/integration.js

```diff
@@ -13,7 +13,7 @@ import {
 const state = { configuration };
 
 describe('Integration tests', () => {
-  describe.skip('bulk', () => {
+  describe('bulk', () => {
```

Comment:
Are these tests supposed to be restored in this PR? 

They don't run in CI right, just locally? So it's probably OK to enable them. I'm just a bit surprised at the diff (and conscious that you're looking at tests in another PR)

Reply:
Yes the don't run in CI, i enabled them to make sure test is passing. They should be enabled